examples/FasterTransformer_HuggingFace_Bert/Bert_FT_trace.py

```diff
@@ -0,0 +1,156 @@
+import argparse
```

Comment:
Just double checking s this mostly your code? If it's heavily adapted from the Faster Transformer codebase. Make sure to add a link to the top saying where this is coming from. Otherwise just resolve this comment.